chert/core.py

```diff
@@ -1066,10 +1071,16 @@ def export_entry(entry):
                 rec.failure('no uploads directory at {}', self.uploads_path)
             else:
                 message = None
-                if os.path.islink(uploads_link_path):
-                    os.unlink(uploads_link_path)
-                    message = 'refreshed existing uploads symlink'
-                os.symlink(self.uploads_path, uploads_link_path)
+                if os.name == "nt":
+                    if os.path.exists(uploads_link_path):
+                        shutil.rmtree(uploads_link_path)
+                    shutil.copytree(self.uploads_path, uploads_link_path)
+                    message = 'refreshed existing uploads files'
```

Comment:
According to [this post](https://stackoverflow.com/questions/26787872/how-to-create-symlinks-in-windows-using-python), creating symlinks is difficult and may require configuration changes that would need to be documented in the README. This workaround just copies the files recursively.

Reply:
If recursive copy works for you on windows, I am fine with that :+1: